applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -453,38 +556,41 @@ int le_audio_pause(void)
 	return 0;
 }
 
-int le_audio_send(uint8_t const *const data, size_t size)
+int iso_stream_send(uint8_t const *const data, size_t size, uint8_t iso_chan_idx)
 {
 	int ret;
 	static bool wrn_printed[CONFIG_BT_ISO_MAX_CHAN];
 	struct net_buf *buf;
 
-	if (is_iso_buffer_full(0)) {
-		if (!wrn_printed[0]) {
+	if (audio_streams[iso_chan_idx].ep->status.state != BT_AUDIO_EP_STATE_STREAMING) {
+		LOG_DBG("channel not connected");
+		return 0;
+	}
+
+	if (is_iso_buffer_full(iso_chan_idx)) {
+		if (!wrn_printed[iso_chan_idx]) {
 			LOG_WRN("HCI ISO TX overrun on ch %d - Single print", 0);
```

Comment:
0 = iso_chan_idx here

Reply:
LOG_WRN("HCI ISO TX overrun on ch %d - Single print", iso_chan_idx);

Reply:
Hmmm, this is a big missed :(